test/range_access.cpp

```diff
@@ -130,15 +130,15 @@ namespace begin_testing {
 		static_assert(!CanBegin<const A>);
 		static_assert(ranges::same_as<decltype(ranges::begin(std::declval<const A&>())), const int*>);
 
-		// Valid: Both member and non-member begin, but non-member returns non-Iterator.
+		// Valid: Both member and non-member begin, but non-member returns non-input_or_output_iterator.
```

Comment:
Seeing usage like this doesn't make me hate the name `input_or_output_iterator` any less. (No change requested.)

Reply:
On the bright side, I wasn't first author on P1754 so future C++ archaeologists probably won't blame me for this name.